crud-frontend/src/App.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { lazy, Suspense, FC } from "react";
+import { Route, Routes, BrowserRouter } from "react-router-dom";
+import LoggedIn from "./pages/loggedIn/loggedIn";
+import HomePage from "./pages/home/home"
+import { AuthProvider } from "./contexts/AuthContext";
+
+// const HomePage = lazy(() => import("./pages/home/home"));
+
+const AppRoutes: FC = () => {
+  return (
+    <AuthProvider>
+      <BrowserRouter>
+        <Suspense fallback={<div>Loading...</div>}>
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/logged-in" element={<LoggedIn />} />
```

Comment:
Normally the URL wouldn't change when you're logged in to a value like this. Was this for experimentation, or you intend to keep it?